app/src/main/res/values/strings.xml

```diff
@@ -330,7 +332,7 @@
     <string name="education_donation_title">Participate</string>
     <string name="education_donation_summary">Participate to help increase the stability, security and in adding more features.</string>
 
-    <string name="html_text_ad_free">Unlike many password management apps, this one is &lt;strong&gt;ad-free&lt;/strong&gt;, &lt;strong&gt;copylefted libre software&lt;/strong&gt; and does not collect personal data on its servers, even in its free version.</string>
+    <string name="html_text_ad_free">Unlike many password management apps, this one is &lt;strong&gt;ad-free&lt;/strong&gt;, &lt;strong&gt;copylefted libre software&lt;/strong&gt; and does not collect personal data on its servers, even in its gratis version.</string>
```

Comment:
There is no "gratis" version.
Only : 
- free
- libre
- pro
(one by flavor in gradle)

Reply:
https://www.keepassdx.com/FAQ.html